### PR TITLE
Update `to_project_space/1` to inject url and cron expression

### LINKED ISSUE #394: Bring Lightning up to date with latest changes to workflow-diagram
Cut a release to npm of kit for workflow-diagram module

Make lightning use latest version

### PR DESCRIPTION
**Pull request by elias-ba**
## Any notes for the reviewer ?
@stuartc this PR depends on the one in kit ([#66](https://github.com/OpenFn/kit/pull/66)). Once that one is merged we will:

- [ ] cut a release on kit and push a new version to [npmjs.org](https://npmjs.org)
- [ ] update the version of kit being used in lightning (in this PR) to test the changes
- [ ] if all good, review and merge this pr.

## Related issue

Fixes #394 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If needed, I've updated the changelog
- [ ] Amber has QA'd this feature

### CODE REVIEW

**Comment by elias-ba** [comment]
@stuartc this PR depends on the one in kit ([#66](https://github.com/OpenFn/kit/pull/66)). Once that one is merged we will:

- [ ] cut a release on kit and push a new version to [npmjs.org](https://npmjs.org)
- [ ] update the version of kit being used in lightning (in this PR) to test the changes
- [ ] if all good, review and merge this pr.

**Review by stuartc — APPROVED** [review]
All good here @elias-ba, fyi I did drop an alias that crept into `lib/lightning/workflows.ex` (`Quantum.Job`, and `...Jobs.Job`).

I've rebased and force-pushed, waiting for CI and I'll merge this. :+1:

**Pull request merged**
